scripts/build.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+if [[ "$(git tag --points-at HEAD 2>/dev/null)"=="v*" ]]; then
```

Comment:
I think you have to add spaces around `==` to make this work.

Reply:
Yep, you are right. And also it should v* instead of "v*", sorry about that too. Fixed now.